nxdlTypes.xsd

```diff
@@ -102,6 +103,34 @@
 		<xs:restriction base="xs:string" />
 	</xs:simpleType>
 	
+	<xs:simpleType name="NX_ARBITRARY_UNITS">
+		<xs:annotation>
+			<xs:documentation>
+				units that cannot be converted into recognized engineering units
+
+				One consideration of arbitrary units arises with software that
+				automatically converts data into some canonical units used by
+				that software. The software should not convert values marked
+				with arbitrary units.
+
+				A second consideration is that when arbitrary units are used,
+				the set of possible analytical results is restricted. In some
+				cases, it is possible to apply a factor to convert the arbitrary
+				units to some engineering units.  This should be considered as a
+				possibility in the analysis process.
+
+				**NOTE**: not the same as NX_UNITLESS or NX_DIMENSIONLESS
+
+				**NOTE**: It is common (such as in :ref:`NXcanSAS`) for intensity to
+				be reported in arbitrary units.
+
+				<xs:element name="example">arbitrary</xs:element>
+				<xs:element name="example">a.u.</xs:element>
```

Comment:
Not to be confused with astronomical units(!)

Reply:
noted